### PR TITLE
Virtual joystick tutorial update

### PR DESCRIPTION
**Pull request by Hackerman342**

### CODE REVIEW

File: website/docs/tutorials/virtual_joystick/03_virtual_joystick_widget.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+By building the `.kv` definition of the `VirtualJoystickWidget` in the Python constructor,
+the widget can be imported just like a kivy API widget.
+That means you can import it into your Python definition of your `App` (i.e., `main.py`), and reference it both there and in your kivy app definition (i.e., `main.kv`)
+
+Explore [**kivy `Builder`**](https://kivy.org/doc/stable/api-kivy.lang.builder.html) for more details.
+
+#### kivy `Clock`
+
+We schedule regular updates to the linked variables containing the pose of the drawn joystick to using the [**kivy Clock**](https://kivy.org/doc/stable/api-kivy.clock.html).
+kivy provides multiple options for scheduling tasks
```
Comment (Hackerman342) [inline website/docs/tutorials/virtual_joystick/03_virtual_joystick_widget.md:42]:
```suggestion
We schedule regular updates to the linked variables containing the pose of the drawn joystick using the [**kivy Clock**](https://kivy.org/doc/stable/api-kivy.clock.html).
kivy provides multiple options for scheduling tasks, which you can explore in their API.
```

Comment (Hackerman342) [inline website/docs/tutorials/virtual_joystick/03_virtual_joystick_widget.md:42]:
```suggestion
We schedule regular updates to the linked variables containing the pose of the drawn joystick using the [**kivy Clock**](https://kivy.org/doc/stable/api-kivy.clock.html).
kivy provides multiple options for scheduling tasks, which you can explore in their API.
```

File: website/docs/tutorials/virtual_joystick/03_virtual_joystick_widget.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+:::
+
+#### Touch handling
+
+The [`on_touch_down()`](https://kivy.org/doc/stable/api-kivy.uix.widget.html#kivy.uix.widget.Widget.on_touch_down), [`on_touch_move()`](https://kivy.org/doc/stable/api-kivy.uix.widget.html#kivy.uix.widget.Widget.on_touch_move), & [`on_touch_up()`](https://kivy.org/doc/stable/api-kivy.uix.widget.html#kivy.uix.widget.Widget.on_touch_up) calls are triggered for all `Widget`s within a kivy `App` whenever there is a touch interaction (by default).
+
+We overwrite the default behavior of these methods to move the pose of the joystick whenever we touch and/or move within the `VirtualJoystickWidget`,
+and recenter the joystick upon release.
+
+From the [**kivy `Widget` class**](https://kivy.org/doc/stable/api-kivy.uix.widget.html), which all widgets inherit from:
```
Comment (Hackerman342) [inline website/docs/tutorials/virtual_joystick/03_virtual_joystick_widget.md:62]:
```suggestion
From the [**kivy `Widget` class docs**](https://kivy.org/doc/stable/api-kivy.uix.widget.html), which all widgets inherit from:
```

**Review by Hackerman342 — COMMENTED** [review]
Minor tweaks

**Review by edgarriba — APPROVED** [review]

**Pull request merged**